pylana/shiny_dashboards.py

```diff
@@ -76,11 +78,12 @@ def upload_shiny_dashboard(self, dashboard_id: str, file: io.IOBase, **kwargs) -
 
         Args:
             dashboard_id: a string denoting the shiny dashboard
-            file: a io.Base pointing to the zipped dashboard code
+            file_path: a io.Base pointing to the zipped dashboard code
```

Comment:
It's the path itself rather than an `io.Base` now.

Reply:
changed in the latest commit